.github/workflows/pull-request.yml

```diff
@@ -78,3 +87,17 @@ jobs:
       - name: Test all links
         run: npm run test:links
 
+  lint:
+    name: Run linter
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - name: Install node dependencies
+        run: npm install
+      - name: Run eslint
+        run: npm run lint
```

Comment:
[clarifying question] Does running eslint also run prettier?

Reply:
No, it does not also run prettier. My thinking was that prettier should be an editor integration and/or manually run on the programmer's end. I don't think we'd want to run it automatically since it doesn't have the kind of comprehensive error checking that eslint does. The purpose of adding a config here was just to ensure that it doesn't clash with eslint. That said, I've only ever used it as an editor/IDE integration and am totally open to other workflows

Reply:
I've seen it as a pre-commit hook in other 18F repos. We can't configure hooks directly in repos, but we can encourage people to use it by adding a template and guidance to the repo. Here's an [example](https://github.com/18F/rails-template/blob/main/templates/githooks/pre-commit)

Reply:
My only concern was making sure people know Prettier is there and should be used, so if other docs accomplish that, then that's sufficient.

Reply:
Let's do both!

Reply:
lovely, thanks!